upoi/map/models.py

```diff
@@ -1,3 +1,70 @@
 from django.db import models
 
-# Create your models here.
+
+class User(models.Model):
```

Comment:
I think it's better to use the User model that Django already has and extend it by using this: https://docs.djangoproject.com/en/1.10/topics/auth/customizing/#extending-the-existing-user-model